pkg/asset/installconfig/gcp/session.go

```diff
@@ -85,31 +94,49 @@ type credLoader interface {
 }
 
 type envLoader struct {
-	env string
+	env      string
+	delegate credLoader
```

Comment:
can we move this change to previous commit.. this doesn't seem to belong to log only once commit..?

Reply:
Good catch, moved.